index.js

```diff
@@ -0,0 +1,31 @@
+const babel = require('babel-core');
+const { types } = babel;
+
+module.exports = function() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.source.value === '@glimmer/component') {
+          let hasTrackedImport = false;
+          path.node.specifiers = path.node.specifiers.filter(specifier => {
+            if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'tracked') {
+              hasTrackedImport = true;
```

Comment:
You're going to want to hoist the matched specifier and then use it below. Consider:

```js
import Component, { tracked as dataStalker } from '@glimmer/component';

export class extends Component {
  @dataStalker whatAWeirdName;
}
```

If you don't hoist the entire specifier you may run into issues where you've created shadowing.

Reply:
Good call